.github/workflows/proto.yml

```diff
@@ -0,0 +1,24 @@
+name: Protobuf Checks
```

Comment:
note to self to look into ways to re-use workflow jobs across workflows (e.g. being able to reference this proto check from the ci job versus having to copy and paste it into the ci workflow)